esp32c2-hal/Cargo.toml

```diff
@@ -38,9 +38,12 @@ embassy-executor  = { package = "embassy-executor", git = "https://github.com/em
 embedded-graphics = "0.7.1"
 esp-backtrace     = { version = "0.6.0", features = ["esp32c2", "panic-handler", "exception-handler", "print-uart"] }
 esp-println       = { version = "0.4.0", features = ["esp32c2"] }
+sha1              = { version = "0.10", default-features = false}
 sha2              = { version = "0.10.6", default-features = false}
 ssd1306           = "0.7.1"
 static_cell       = "1.0.0"
+hex-literal       = "0.4"
+p256              = { version = "0.13", default-features = false }
```

Comment:
I don't think these are being used.

Reply:
You're right! I've started coding for ECC and these lines are not necessary for HMAC :).
I'll update the file to remove them.